simespresso/io_QE/tests/test_start_espresso.py

```diff
@@ -0,0 +1,24 @@
+
+import unittest
+import os
+from simespresso.io_QE import espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
```

Comment:
> ok so I guess I need to wait for this to be implemented? 

@jeremy-rutman , you will have to implement a concrete class of 'ABCModelingEngine'   (@ahashibon )